support-frontend/assets/pages/digital-subscription-checkout/components/checkoutForm.jsx

```diff
@@ -160,6 +143,15 @@ function CheckoutForm(props: PropTypes) {
       billingPeriod={Annual}
     />) : '';
 
+  const isPayPalEnabled = (optimizeExperiments: OptimizeExperiments) => {
+    const PayPalExperimentId = '8IebFnX-SbKRxhlj_tGp-w';
+    const enabledByTest = optimizeExperiments.find(exp => exp.id === PayPalExperimentId && exp.variant === '1');
+    const enabledByQueryString = getQueryParameter('payPal') === 'true';
+    return enabledByTest || enabledByQueryString;
+  };
+
+  const payPalEnabled = isPayPalEnabled(props.optimizeExperiments);
```

Comment:
Would be good to pass `payPalEnabled` into AddressForm instead of `optimizeExperiments` as we've already worked it out here.